src/List.module.css

```diff
@@ -0,0 +1,18 @@
+.list-container {
+  height: 50vh;
+  width: 25vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.list-results-container {
+  height: 66%;
+  width: 100%;
+  overflow-y: scroll;
+}
+
+.ul-list {
+  text-align: left;
```

Comment:
maybe add margin-top: 0;
 to tighten up the space a bit.
![image](https://user-images.githubusercontent.com/10378331/91499873-e6491680-e887-11ea-9560-28e5a634f69c.png)

vs.:
![image](https://user-images.githubusercontent.com/10378331/91499907-f8c35000-e887-11ea-9cc9-98aeadb5deb5.png)